pandas/core/generic.py

```diff
@@ -4871,7 +4872,7 @@ def describe_1d(data, percentiles):
                     names.append(name)
         d = pd.concat(ldesc, join_axes=pd.Index([names]), axis=1)
         d.columns.names = data.columns.names
-        return d
+        return self._constructor(d)
```

Comment:
this results in additional construction overhead. Better to add some logic to ``concat`` to enable this